src/app/modules/project/overview/overview.component.ts

```diff
@@ -84,7 +86,9 @@ export class OverviewComponent implements OnInit, AfterViewInit {
       // We need the 1ms timeout to make sure the components are rendered, I don't know why angular doesn't do this but whatever
       setTimeout(() => {
         this.amountOfProjectsOnSinglePage = this.filterMenu.amountOfProjectsOnSinglePage;
-        this.projectList.createProjectModal(projectId);
+        if (projectId) {
+          this.modalUtility.openProjectModal(projectId, '' , '/project/overview');
```

Comment:
This should work, might be smart to add a check if the project with `projectId` is present on the current page because then it needs to subscribe to the changes.

Reply:
I tried to fix this in my last commit, but it was a lot more difficult then I thought due to the modal being loaded before the overview in the background. 

If you'd like to look at it that would be amazing, please let me know if you think this would be allright or if you know a more sophisticated solution 😸

Reply:
Your solution should work fine, if you want to get rid of the interval you could add an [output](https://angular.io/guide/inputs-outputs#sending-data-to-a-parent-component) to the project-list component that sends all the loaded projectIds to the overview component. When the projects are done loading, check if the current project is in the list and subscribe to the likesif needed.